src/deluge/model/sample/sample.h

```diff
@@ -140,6 +143,8 @@ class Sample final : public AudioFile {
 
 	SampleClusterArray clusters;
 
+	AbsValueFollower envelopeFollower{};
```

Comment:
How big are these? We dont really need one per sample, might be better to just allocate one when record is activated. Maybe even just have a single one that's always re used for threshold sampling?